templates/bands-development.ini.erb

```diff
@@ -0,0 +1,9 @@
+# THIS FILE IS GENERATED BY PUPPET
+# DO NOT EDIT DIRECTLY, INSTEAD EDIT THE .erb template
+
+
+
+[dates]
+<% if @stage_agreement_deadline %>
+stage_agreement_deadline = <%= @stage_agreement_deadline %>
+<% end -%>
```

Comment:
there's a lot more to add here to get this working, but it's a good starting point